jest-puppeteer.config.js

```diff
@@ -1,4 +1,8 @@
 module.exports = {
+  launch: {
+    args: ['--no-sandbox', '--disable-setuid-sandbox'],
```

Comment:
Is it maybe `--disable-setup-sandbox`?

Reply:
No it's correct: https://github.com/puppeteer/puppeteer/blob/master/docs/troubleshooting.md#setting-up-chrome-linux-sandbox